huaweicloud/services/dms/resource_huaweicloud_dms_rocketmq_instance.go

```diff
@@ -416,7 +417,7 @@ func resourceDmsRocketMQInstanceUpdate(ctx context.Context, d *schema.ResourceDa
 	return resourceDmsRocketMQInstanceRead(ctx, d, meta)
 }
 
-func buildUpdateRocketmqInstanceBodyParams(d *schema.ResourceData, config *config.Config) map[string]interface{} {
+func buildUpdateRocketmqInstanceBodyParams(d *schema.ResourceData, _ *config.Config) map[string]interface{} {
```

Comment:
If the config is not used, we should remove it from the arguemnt list.

Reply:
done